ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -160,18 +154,26 @@ def getPullFeatures(self, pull):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
```

Comment:
The description should cover both cases- single repo & all repos

Reply:
Done.